extension/src/popup/gql/index.js

```diff
@@ -15,7 +15,7 @@ export const Networks = gql`
       chain_id
       testnet
       title
-      rpc_url
+      public_rpc_url
```

Comment:
To be honest, I still don't quite understand the why for having `public_rpc_url`, but it's OK

Reply:
the internal rpc url is blocked for users.

Reply:
oh cool